pages/api/workouts/[id].js

```diff
@@ -0,0 +1,30 @@
+import dbConnect from "../../../db/dbConnect";
+import Workout from "../../../db/models/Workouts";
+
+// /api/questions/id
+export default async function handler(req, res) {
+  await dbConnect();
+
+  const { id } = req.query;
+
+  if (req.method === "GET") {
+    try {
+      const workout = await Workout.findById(id);
+
+      res.status(200).json({
+        id: workout._id,
+        name: workout.name,
+      });
```

Comment:
wenn das hier noch komplizierter wird später, würde ich das Objekt noch in einer Konstante speichern, glaub ich